src/experimental-layers/src/path-marker-layer/create-path-markers.js

```diff
@@ -39,9 +37,20 @@ export default function createPathMarkers({
 
     // Create the markers
     for (const percentage of percentages) {
-      const marker = createMarkerAlongPath(
-        {path: vPoints, percentage, lineLength, color, object, projectFlat});
-      markers.push(marker);
+
+      if (direction.forward) {
+        const marker = createMarkerAlongPath(
+          {path: vPoints, percentage, lineLength, color, object, projectFlat});
+        markers.push(marker);
+      }
+
+      if (direction.backward) {
+        vPoints.reverse();
```

Comment:
This will cause an issue on multiple percentages.

Reply:
Maybe leave a comment/TODO?

Reply:
I have a fix.